tlf-baseline.Rmd

```diff
@@ -75,10 +75,10 @@ tbl_base
 We define the format of the output. We highlight items that
 are not discussed in previous discussion.
 
-`text_indent_first` and `test_indent_left` are used to control the indent
+`text_indent_first` and `text_indent_left` are used to control the indent
 space of text. They are helpful when you need to control the white space
 of a long phrase "AMERICAN INDIAN OR ALASKA NATIVE"
-in the table provide an example.
+in the table provides an example.
```

Comment:
I'm confused by this sentence. Should it be two separate sentences instead?

Reply:
yeah, a little bit confusing. I added a ',' after the wording 'phrase', is this one better?